README.md

```diff
@@ -1,8 +1,43 @@
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Braikhna Yousafzai . Jarvis Consulting
 
+I enjoy the growth and constant learning that comes with the software industry. I am looking to apply and grow my development and engineering skills in creative ways. I enjoy working collaboratively and across disciplines to get a comprehensive understanding of my projects and company. I appreciate team-based approaches towards a cohesive solution. I am a recent graduate from Western University with a Bachelor's degree with Specialization in Computer Science. I have worked on several Java, Python, and database management projects throughout my degree. I most enjoy the engineering, data management, and analytic aspects of the industry. Outside of work, I enjoy painting, coffee, and a good book. I am also quite involved with my community in volunteering and leadership capacities.
+
+## Skills
+
+**Proficient:** Java, RDBMS/SQL, Agile/Scrum, Linux/Bash, Git, C/C++
+
+**Competent:** Python, R, Docker, PHP, HTML, CSS
+
+**Familiar:** Hadoop, NoSQL, Tableau, Crystal Reports, Scala
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_BraikhnaYousafzai/tree/master/linux_sql)]: Implemented a linux cluster monitoring agent using shell scripts inserting hardware and usage data for each cluster node to a database. The data was collected in real time using a Crontab job and persisted using PostgreSQL. The project was deployed using a docker container and version control was maintained through GitHub.
+
+
+## Highlighted Projects
+**COVID-19 Graphing Program** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/A1)]: This project pulls Covid-19 tracking data from an API and sorts the data in ascending or descending order depending on user input. It graphs the data to show various trends such as countries with highest number of total cases or countries with highest number of deaths.
+
+**COVID 19 Contact Tracing System** [[GitHub](https://github.com/bmaqs/CS3307-Assignments/tree/master/Group-45-Project)]: Created a software allowing businesses to track incidences of COVID-19 in their establishments and automates notifications to their customer base of any possible exposures. It also allows consumers to easily report COVID-19 cases they have experienced as well as easily receive notifications about potential exposures.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Worked on development and engineering projects in an agile enviornment. Worked on a scrum team, including taking on duties as a scrum master.
```

Comment:
Worked on development and engineering projects in an agile environment. Worked on a scrum team, including taking on duties as a scrum master.

Responsbilities 
soft skills, team, coll, comm, prob, leader
agile